AppManager/app_manager_server.py

```diff
@@ -673,6 +679,26 @@ def copy_modified_jars(app_name):
     logging.error("Failed to copy email jars to lib directory of " + app_name)
     return False
 
+  cp_mr_jar = subprocess.call("cp " + appscale_home + "/ext/java/" +\
+    "appscale-mapreduce-0.8.5.jar " + lib_dir, shell=True)
```

Comment:
Yeah it should be variable with the releases. I will change that to parse from the pom file